js/typechain/factories/ERC1967UpgradeUpgradeable__factory.js

```diff
@@ -0,0 +1,64 @@
+"use strict";
```

Comment:
Is it possible to avoid checking in `typechain` and have it ignored in git/generated on the fly?

Reply:
I'm not familiar with npm publish, but if you think removing these types would still allow the library to work, then yeah, we could ignore it. 

As for generated on the fly, how would I do that given the typechain is generated in another repo? Shoot me some pointers and I'm ready to look at it.